doc/api/http.md

```diff
@@ -3720,9 +3720,11 @@ events will be emitted in the following order:
 Setting the `timeout` option or using the `setTimeout()` function will
 not abort the request or do anything besides add a `'timeout'` event.
 
-Passing an `AbortSignal` and then calling `abort` on the corresponding
+Passing an `AbortSignal` and then calling `abort()` on the corresponding
 `AbortController` will behave the same way as calling `.destroy()` on the
-request itself.
+request. Specifically, the `'error'` event will be emitted with an error with
+the message `'AbortError: The operation was aborted'`, the code `'ABORT_ERR'`
```

Comment:
```suggestion
request. Specifically, the `'error'` event will be emitted with the error message `'AbortError: The operation was aborted'`, the code `'ABORT_ERR'`
```

Reply:
This parallels the wording in the bullet points above and makes it clear that the argument is an Error and not a string.

Reply:
It sounds a bit reduntant to me, feel free to ignore